ui/app/components/app/gas-customization/gas-price-chart/index.scss

```diff
@@ -52,10 +52,9 @@
   }
 
   .custom-tooltip th {
-    font-style: normal;
+    @include H8;
+
     font-weight: 500;
-    line-height: normal;
```

Comment:
The line height is changing here but it appears to have zero impact, since the tooltip has a fixed height.